docs.openc3.com/src/theme/DocCard/index.js

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import clsx from "clsx";
+import Link from "@docusaurus/Link";
+import {
+  useDocById,
+  findFirstSidebarItemLink,
+} from "@docusaurus/plugin-content-docs/client";
+import { usePluralForm } from "@docusaurus/theme-common";
+import isInternalUrl from "@docusaurus/isInternalUrl";
+import { translate } from "@docusaurus/Translate";
+import Heading from "@theme/Heading";
+import styles from "./styles.module.css";
+function useCategoryItemsPlural() {
+  const { selectMessage } = usePluralForm();
+  return (count) =>
+    selectMessage(
+      count,
+      translate(
+        {
+          message: "1 item|{count} items",
+          id: "theme.docs.DocCard.categoryDescription.plurals",
+          description:
+            "The default description for a category card in the generated index about how many items this category includes",
+        },
+        { count }
+      )
+    );
+}
+function CardContainer({ href, children }) {
+  return (
+    <Link
+      href={href}
+      className={clsx("card padding--lg", styles.cardContainer)}
+    >
+      {children}
+    </Link>
+  );
+}
+function CardLayout({ href, icon, title, description }) {
+  return (
+    <CardContainer href={href}>
+      <Heading
+        as="h2"
+        className={clsx("text--truncate", styles.cardTitle)}
+        title={title}
+      >
+        {icon} {title}
+      </Heading>
+      {description && (
+        <p
+          className={clsx("text--truncate", styles.cardDescription)}
+          title={description}
+        >
+          {description}
+        </p>
+      )}
+    </CardContainer>
+  );
+}
+function CardCategory({ item }) {
+  const href = findFirstSidebarItemLink(item);
+  const categoryItemsPlural = useCategoryItemsPlural();
+  // Unexpected: categories that don't have a link have been filtered upfront
+  if (!href) {
+    return null;
+  }
+  return (
+    <CardLayout
+      href={href}
+      icon="🗃️"
+      title={item.label}
+      description={item.description ?? categoryItemsPlural(item.items.length)}
+    />
+  );
+}
+function CardLink({ item }) {
+  const icon =
+    item?.customProps?.myEmoji ?? (isInternalUrl(item.href) ? "📄️" : "🔗");
```

Comment:
This is the only line that I changed after 'swizzeling` the DocCard. I followed this example: https://github.com/facebook/docusaurus/discussions/10476

```
npm run swizzle @docusaurus/theme-classic DocCard -- --eject
```